app/helpers/application_helper.rb

```diff
@@ -0,0 +1,12 @@
+module ApplicationHelper
+
+  def current_user
+    User.find_by(id: session[:user_id])
```

Comment:
This shouldn't be here. Just mark the current_user in ApplicationController with helper_method :current_user